package.json

```diff
@@ -22,7 +22,8 @@
     "redux": "3.6.0",
     "redux-saga": "0.15.3",
     "reselect": "3.0.1",
-    "whatwg-fetch": "1.1.1"
+    "whatwg-fetch": "1.1.1",
+    "bootstrap": "^3.3.7"
```

Comment:
Moving this from devDependencies to dependencies is correct. It should not be hidden in this commit, though.